.clang-format

```diff
@@ -1,23 +1,109 @@
-BasedOnStyle: LLVM
-IndentWidth: 4
-SortIncludes:    false
-ColumnLimit:     120
-AlignTrailingComments: false
-AccessModifierOffset: -4
-AlignConsecutiveAssignments: true
-ReflowComments:  false
-BraceWrapping:   
-  AfterClass:    true
-  AfterFunction: true
-  BeforeElse: true
-  BeforeCatch: true
-  AfterNamespace:  true
-  AfterEnum: true
-BreakBeforeBraces: "Custom"
-PointerAlignment: Left
-AllowShortFunctionsOnASingleLine: false
-NamespaceIndentation: Inner
+---
+Language: Cpp
```

Comment:
@julianlitz Can you plase say why you introduced new coding styles and what they change?

Reply:
Thanks for catching that I experimented with various clang-format settings to see what would work best, which may have caused some unexpected changes. We can definitely revert back to the previous style if needed.

That said, I’d still recommend two specific changes:

Set AlignConsecutiveAssignments: true to false
This will reduce alignment for consecutive assignments, which I believe improves readability and avoids large shifts in code alignment.

Increase the ColumnLimit
A higher column limit allows more code per line, which can help reduce unnecessary line breaks and make the code easier to scan.

Reply:
Could you give me an example for the first? For the second I'd like to keep the 120.

Reply:
![image](https://github.com/user-attachments/assets/6759558b-9a9e-4a70-9df4-1f41656d15eb)